docs/openapi.yaml

```diff
@@ -20,11 +20,12 @@ info:
     For information regarding authentication please read [our guide](https://github.com/vippsas/vipps-recurring-api/blob/master/vipps-recurring-api.md#authentication)
 
 
-  version: 2.5.3
+  version: 2.5.4
```

Comment:
Shouldn't this be considered something more like '2.6.0' at the least? 
We are adding a new version after all.